native/explorer/src/lib.rs

```diff
@@ -280,6 +296,7 @@ rustler::init!(
         s_slice_by_indices,
         s_sort,
         s_standard_deviation,
+        s_strip,
```

Comment:
I also think we have been naming those functions after the Elixir side. I.e. we should let Elixir dictates the names as much as possible. :)

Reply:
Ah really? My mental model was the opposite, but I'm not attached to it at all. :) Happy to go that way.

Reply:
For sure in the Backend we want to have Elixir specific names. Here whatever name is fine but it seems we are mostly aligned to the Elixir side? :)